### PR TITLE
Issue #2526 - Fix Tolerance SerializationException in low-trust tests

### PR DESCRIPTION
**Pull request by ggeurts**

### CODE REVIEW

File: src/NUnitFramework/framework/Constraints/Tolerance.cs [hunk 1]
```diff
@@ -32,6 +32,9 @@ namespace NUnit.Framework.Constraints
     /// type that supports taking a difference between two 
     /// objects and comparing that difference to a value.
     /// </summary>
+#if !NETSTANDARD1_3 && !NETSTANDARD1_6
+    //[Serializable]
```
Comment (CharliePoole) [inline src/NUnitFramework/framework/Constraints/Tolerance.cs]: I can't imagine this is what you intended. 😸

Comment (ggeurts) [reply src/NUnitFramework/framework/Constraints/Tolerance.cs]: Hmmm, neither can I....

Comment (ggeurts) [reply src/NUnitFramework/framework/Constraints/Tolerance.cs]: Commit has been amended.

**Comment by CharliePoole** [comment]
Spoke too soon! Your new test is throwing a `NullReferenceException` in Travis-CI.

**Comment by rprouse** [comment]
Very odd, I can't see anything about the changed code that should cause a `NullReferenceException`. The older version of Mono also passes which is more confusing.

**Comment by ggeurts** [comment]
The new test didn't exclude Mono as platform, which it should do as Mono doesn't implement code access security.

**Comment by rprouse** [comment]
Thanks for the contribution!

**Pull request merged**